packages/edge-login-ui-rn/src/types/Theme.js

```diff
@@ -6,6 +6,10 @@ export type Theme = {|
 
   pressedOpacity: number,
   fontFamily: string,
+  fontWeightBold: string,
+
+  // Icons
+  icon: string,
```

Comment:
The `icon` and `warningText` styles are wrong here. They should be `dropdownWarning`, `dropdownError`, and `dropdownText` which would be used for both the icons and the text.

Reply:
Fixed